charts/redpanda/values.yaml

```diff
@@ -550,7 +550,22 @@ config:
     # tm_sync_timeout_ms: 2000ms                                   # Time to wait state catch up before rejecting a request
     # tm_violation_recovery_policy: crash                          # Describes how to recover from an invariant violation happened on the transaction coordinator level
     # transactional_id_expiration_ms: 10080min                     # Producer ids are expired once this time has elapsed after the last write with the given producer ID
-  tunable: {}
+  tunable:
+    log_segment_size: 134217728                                    # 128 mb
+    log_segment_size_min: 16777216                                 # 16 mb
+    log_segment_size_max: 268435456                                # 256 mb
+    kafka_batch_max_bytes: 1048576                                 # 1 mb
+    topic_partitions_per_shard: 1000
+    compacted_log_segment_size: 67108864                           # 64 mb
+    max_compacted_log_segment_size: 536870912                      # 512 mb
+    kafka_connections_max: 15100
```

Comment:
Setting a connection limit probably doesn't make sense, if you don't know how large the machines you're installing onto are.

Reply:
Nope.  In cloud SaaS contexts, the connection count is part of the definition of the product, we do not have a matrix of how many connections a given self-hosted machine type can handle (with memory limits, it's hard to define, because a system with lots of partitions can handle fewer clients, etc, so it depends on the workload).

The default for self hosted clusters today is not to apply a connection  count limit.